pro/src/pages/IndividualOffer/IndividualOfferConfirmation/IndividualOfferConfirmation.spec.tsx

```diff
@@ -102,9 +102,30 @@ describe('IndividualOfferConfirmation', () => {
     ).toHaveAttribute('href', `/offres`)
   })
 
+  it('should display a pending message when offer is scheduled and pending for validation', () => {
+    offer.publicationDate = new Date(Date.now() + 3600).toISOString()
+    offer.status = OfferStatus.PENDING
+    renderOffer(contextOverride)
+    expect(screen.getByText('Offre programmée en cours de validation')).toBeInTheDocument()
+    expect(
+      screen.queryByText('Visualiser l’offre dans l’application', {
+        selector: 'a',
+      })
```

Comment:
À titre informatif, on préfère généralement utiliser des getters par rôle ARIA :
```suggestion
screen.getByRole('link', { name: 'Créer une nouvelle offre' })
```